api/test/robot_tests/cmd/keywords.resource

```diff
@@ -7,7 +7,7 @@ Resource            ../keywords.resource
 Resource            ../variables.resource
 
 *** Variables ***
-${WALLET_PLUGIN_VERSION}       v0.3.0
+${WALLET_PLUGIN_VERSION}       v0.3.5
```

Comment:
v0.3.5 is a prerelease, nothing change for the end user, only the env prompter is updated with the minimal fees, here we use this version to be able to submit op in the buildnet.